packages/angular/src/tracing.ts

```diff
@@ -61,22 +61,23 @@ export class TraceService {
       }
 
       const navigationEvent = event as NavigationStart;
+      const strippedUrl = stripUrlQueryAndFragment(navigationEvent.url);
       let activeTransaction = getActiveTransaction();
 
       if (!activeTransaction && stashedStartTransactionOnLocationChange) {
         activeTransaction = stashedStartTransaction({
-          name: navigationEvent.url,
+          name: strippedUrl,
           op: 'navigation',
         });
       }
 
       if (activeTransaction) {
         this._routingSpan = activeTransaction.startChild({
-          description: `${navigationEvent.url}`,
+          description: `${strippedUrl}`,
```

Comment:
Since we have already the stripped URL in the transaction name and the tag, would it make sense to keep the full URL here?

Reply:
Your call, I'm fine with any cc @HazAT

Reply:
yeah, the change is to improve transaction grouping having the full URL here just provides more insights. So I am pro Full URL in description